lib/plugins/create/templates/aws-nodejs/serverless.yml

```diff
@@ -101,3 +79,15 @@ functions:
 #     NewOutput:
 #       Description: "Description for the output"
 #       Value: "Some output value"
+
+
+##───────────────────────────────────────────────────────────────────────
+## 'package' - Include or Exclude artifacts in your service http://bit.ly/2rJBhU9
```

Comment:
Same comment about the link. Maybe we could mention that the user can use the git familiar glob syntax here (although this could be an obvious one since the example showcases that).